README.md

```diff
@@ -179,3 +179,32 @@ By default, files are compared after gzip compression, but it's possible to use
 ```yaml
 compression: "none"
 ```
+
+### Checking multiple bundles
+
+The action reuses the same comment each time it runs on a PR. In order to run the action multiple times against separate bundles for a single PR, you must provide a `comment-key` option, which the action will use to determine which comment to add or update for the run. The example below demonstrates this for separate "modern" and "legacy" bundles:
+
+```yaml
```

Comment:
Similar to the other options that have "full config" documentation, let's switch this to a diff to emphasize the option over the other config bits.

```suggestion
```diff
```